apps/src/responsive.jsx

```diff
@@ -0,0 +1,119 @@
+/* A very simple responsive layout system.
+ */
+
+import * as utils from './utils';
+import $ from 'jquery';
+
+export default class Responsive {
+
+  constructor(responsiveWindowWidth) {
+    this.responsiveWindowWidth = responsiveWindowWidth;
+  }
+
+  /**
+   * Gets the container width.
+   * Returns either a number (e.g. 1170) or a string (e.g. "97%").
+   */
+  getResponsiveContainerWidth() {
+    const windowWidth = $(window).width();
+
+    if (windowWidth >= 970+50) {
+      return 970;
+    } else {
+      return "97%";
+      // return Math.round(0.97 * windowWidth);
```

Comment:
what's this comment for?

Reply:
Should we be using `styleConstants['content-width']` instead of 970 here?
Where does the 50 come from?
Why 97%?
(These things could both use comments imo)

Reply:
I'lll add comments and appropriate variables.  The 50px is to give us 25px pixels as left/right magins beside the 970px content container.  The 97% is a different way to ensure left/right margins when the browser is smaller.